extensions/wikia/PortableInfobox/tests/PortableInfoboxRenderServiceHelperTest.php

```diff
@@ -0,0 +1,273 @@
+<?php
+
+use Wikia\PortableInfobox\Helpers\PortableInfoboxRenderServiceHelper;
+
+class PortableInfoboxRenderServiceHelperTest extends WikiaBaseTest {
+	private $helper;
+
+	protected function setUp() {
+		$this->setupFile = dirname( __FILE__ ) . '/../PortableInfobox.setup.php';
+		parent::setUp();
+
+		$this->helper = new PortableInfoboxRenderServiceHelper();
+	}
+
+	/**
+	 * @desc creates mock of PortableInfoboxRenderServiceHelper
+	 * @param array $input
+	 * @return PHPUnit_Framework_MockObject_MockObject
```

Comment:
The description and name fell out of date with the contents. My bad, should be something along the lines of mockWikiaFileHelper